app/models/data-operation.ts

```diff
@@ -852,6 +852,24 @@ export default class DataOperation {
     this.model.stagingLocation = value;
   }
 
+  /**
```

Comment:
If you want to store the UMM collections to use later on, there are generally two ways:

1.) Add it to RequestContext
2.) Add it as a top level property of the Data operation class.

Only fields that get serialized should be added to the model. Also a nit - call it `ummCollections`.